installer.sh

```diff
@@ -44,7 +44,12 @@ function install_service {
     NAME=${CATALOG_RECORD[0]//,}
     PLAN=${CATALOG_RECORD[1]//,}
     echo $NAME $PLAN
-    RETURN=($(cf create-service ${NAME} ${PLAN} "my$1"))
+	if [ -z "$2" ];
```

Comment:
since you grep CATALOG_RECORD, why you need to pass full "postgresql193" name? Doesn't grep 'postgres' returns at index 0 postgresql193 ?